# test: fail
# expect: This should cause a failure code.

fn main() {
    panic("This should cause a failure code.")
}
